<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shp2GpkgDialog</class>
 <widget class="QDialog" name="Shp2GpkgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SHP ‚Üí GPKG</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">

   <!-- Columna izquierda: opciones -->
   <item>
    <layout class="QVBoxLayout" name="leftLayout">

     <!-- Carpeta de entrada -->
     <item>
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>üìÇ Carpeta de entrada (Shapefiles):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="inputLayout">
       <item>
        <widget class="QLineEdit" name="inputFolderLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="inputBrowseButton">
         <property name="text">
          <string>Explorar...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <!-- Carpeta de salida -->
     <item>
      <widget class="QLabel" name="outputLabel">
       <property name="text">
        <string>üíæ Carpeta de salida (GeoPackage):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="outputLayout">
       <item>
        <widget class="QLineEdit" name="outputFolderLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="outputBrowseButton">
         <property name="text">
          <string>Explorar...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <!-- EPSG destino -->
     <item>
      <widget class="QLabel" name="epsgLabelTitle">
       <property name="text">
        <string>üåç EPSG (opcional):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="epsgLayout">
       <item>
        <widget class="QLineEdit" name="epsgLineEdit"/>
       </item>
      </layout>
     </item>

     <!-- Log de ejecuci√≥n -->
     <item>
      <widget class="QLabel" name="logLabel">
       <property name="text">
        <string>üìú Registro de ejecuci√≥n:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="logTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Mensajes de ejecuci√≥n...</string>
       </property>
      </widget>
     </item>

     <!-- Botones -->
     <item>
      <layout class="QHBoxLayout" name="buttonsLayout">
       <item>
        <widget class="QPushButton" name="runButton">
         <property name="text">
          <string>‚ñ∂ Ejecutar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>‚ùå Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

    </layout>
   </item>

   <!-- Columna derecha: descripci√≥n + advertencia -->
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <item>
      <widget class="QTextEdit" name="descriptionText">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>
         <![CDATA[
         <h1><b>shp2gpkg</b></h1>
         <p><b>Descripci√≥n de la herramienta:</b></p>
         <p>Convierte todos los archivos <b>Shapefile (.shp)</b> de la carpeta seleccionada (y de sus subcarpetas internas) en archivos <b>GeoPackage (.gpkg)</b>, 
            conservando la misma estructura de carpetas y generando un archivo resumen con los resultados de las capas transformadas.</p>
         <ul>
           <li><b>Carpeta de entrada:</b> donde est√°n los shapefiles.</li>
           <li><b>Carpeta de salida:</b> destino de los .gpkg.</li>
           <li><b>EPSG (opcional):</b> reproyecta los datos al EPSG indicado.</li>
         </ul>
         <p><b>‚ö† Advertencia:</b> Si no se ingresa un valor de EPSG o se escriben letras, cada capa conservar√° su sistema de referencia original. 
            Las que originamente no tengan un EPSG se les asignar√° el 4326 por defecto. Si se ingresa un n√∫mero EPSG inv√°lido, la conversi√≥n de las capas fallar√°n y el resultado ser√° vac√≠o.</p>
         <p><b>‚ö† Advertencia importante:</b> Si un archivo GPKG ya existe, la capa correspondiente ser√° reemplazada autom√°ticamente.</p>
         ]]>
        </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

